js/sites.js

```diff
@@ -84,10 +85,11 @@
      */
     unpin() {
       if (this.isPinned()) {
-        gNewTab.sendToBrowser("NewTab:UnpinLink", {
+        gNewTab.sendToBrowser("NewTab:GetUpdatePages", {
```

Comment:
Mentioned this in a comment (https://github.com/mozilla/newtab-dev/commit/906f24ce3ff84233c7d3bba07611374e290a2f25#diff-a56c46bbafa8c902880b59d521a047ffR41) but can we name this "NewTab:UpdatePages" for consistency here too